docs/README.md

```diff
@@ -1,28 +1,62 @@
+<h4 align="center"><img src="media/crossplane-overview.png" alt="Crossplane" height="400"></h4>
 
-Crossplane includes a workload scheduler that can factor a number of criteria including capabilities, availability, reliability, cost, regions, and performance while deploying workloads and their resources. The scheduler works alongside specialized resource controllers to ensure policies set by administrators are honored.
+Crossplane has four main feature areas:
+1. Crossplane Services - provision managed services from kubectl.
+1. Crossplane Stacks - extend Crossplane with new functionality.
+1. Crossplane Workloads - define complete applications and schedule across clusters, regions, and clouds.
+1. Crossplane Clusters - manage multiple Kubernetes clusters from a single control plane.
+
+## Crossplane Services
+Crossplane Services supports managed service provisioning using `kubectl`. It applies the Kubernetes pattern for Persistent Volume (PV) claims and classes to managed service provisioning with support for a strong separation of concern between app teams and cluster administrators. 
+
+App teams can choose between cloud-specific and portable services including managed databases, message queues, buckets, data pipelines, and more to define complete applications, build once, and deploy into multiple clouds using continuous delivery pipelines or GitOps flows. 
+
+Cluster administrators can define self-service policies and best-practice configurations to accelerate app delivery and improve security, so app teams can focus on delivering their app instead of cloud-specific infrastructure details.
+
+The [Using Services](using-services.md) guide explores related concepts and shows how to use Crossplane Services to deploy a Wordpress instance that securely consumes a managed `MySQLInstance` from GCP, AWS, or Azure from `kubectl`.
+
+## Crossplane Stacks
+Stacks extend Crossplane with new functionality and can be installed using the [Stack Manager](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md) via the [Kubernetes API](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-stacks.md#installation-flow) or with the [crossplane kubectl plugin](https://github.com/crossplaneio/crossplane-cli#crossplane-cli) [`stack install`](https://github.com/crossplaneio/crossplane-cli#usage) command. Crossplane is built on the Kubernetes API machinery as a platform, and Stacks simplify extending the Kubernetes API to provision and lifecycle manage apps and the managed service infrastructure they depend on, including [Stack security and isolation](https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-stacks-security-isolation.md).
```

Comment:
I would recommend using reference-style links (see [this document](https://daringfireball.net/projects/markdown/syntax#link) for some more information or the bottom of the [stacks guide document](https://github.com/suskin/crossplane/blob/stack-guide/docs/stacks-guide.md)); I find the links are easier to manage that way for long documents.

Reply:
good idea, will look at fixing those